src/components/HomePage/UpcomingEvents.js

```diff
@@ -29,29 +39,45 @@ function UpcomingEvents() {
 	const currentTime = new Date();
 	let eventsDisplayed = 0;
 	const numEventsToDisplay = 2;
-	const timeSlots = [];
+
+	const events = [];
 	// Assumes workshopSchedule is ordered by time
 	for (let i = 0; i < workshopSchedule.length && eventsDisplayed < numEventsToDisplay; i++) {
-		const workshop = workshopSchedule[i];
-		if (currentTime < workshop.startTime) {
-			timeSlots.push(workshop);
-			eventsDisplayed += workshop.events.length;
+		const timeSlot = workshopSchedule[i];
+		if (currentTime < timeSlot.startTime) {
+			timeSlot.events.forEach(event => {
+				const eventFormat = {
+					startTime: `${timeFormatter.format(timeSlot.startTime)} ${currentTimeZoneShort}`,
+					...event
+				};
+				events.push(eventFormat);
+			});
+			eventsDisplayed += timeSlot.events.length;
 		}
 	}
 	if (eventsDisplayed === 0) {
 		return null; // Abort entirely
 	}
-	return <Container component='section' maxWidth='md' className={classes.eventsContainer}>
-		<Grid container justify='center' spacing={1}>
-			<Typography align='center' component='h1' variant='h4'
+	const renderEvents = eventsDisplayed === 0 ?
+		null :
+		events.map((event, index) => {
+			return (
+				<>
+					<Event key={index} {...event} />
+					{index === events.length - 1 ? null : <Divider variant='inset' className={classes.divider} />}
+				</>
+			);
+		});
+
+	return <Box component="section" paddingY={{ xs: 8, md: 10 }} bgcolor='background.grey'>
+		<Container maxWidth='md' className={classes.eventsContainer}>
+			<Typography align='left' component='h1' variant='h4'
```

Comment:
```suggestion
			<Typography component='h2' variant='h4'
```

`align` doesn't seem necessary, and logically this is a level-2 heading of the page, same as other headings like "FAQ".